projects/ad9213_evb/vcu118/README.md

```diff
@@ -0,0 +1,20 @@
+# AD9213/VCU118 HDL Project
+
+## Building the project
+
+```
+cd projects/ad9213_evb/vcu118
+make
+```
+
+The RX link mode can be found in the [AD9213 data sheet](https://www.analog.com/media/en/technical-documentation/data-sheets/AD9213.pdf).
+
+This project is not parameterizable, it has a fixed configuration.
+
+The parameters from the environment:
+
+- RX_NUM_OF_CONVERTERS - RX number of converters per link
```

Comment:
Could you please write the fixed value for each parameter as I wrote it for AD9208? https://github.com/analogdevicesinc/hdl/pull/1726/files#diff-0ad734a165506cbbd282dd7123b268d4dd2db3f972b77b8259ea5e318762ad3b